README.md

```diff
@@ -68,6 +68,27 @@ bimtap.ExportStream(resultExport, AnyBitmap.ImageFormat.Jpeg, 100);
 System.Drawing.Bitmap image = new System.Drawing.Bitmap("FILE_PATH");
 IronSoftware.Drawing.AnyBitmap anyBitmap = image;
 anyBitmap.SaveAs("result-from-casting.png");
+
+
+// Create multiple page Tiff image by Array of AnyBitmap
```

Comment:
Would use: `Creates an AnyBitmap from an array of Tiff images`

Reply:
But the example uses `"FILE_PATH_1"`, `"FILE_PATH_2"`. Should we change them to `"FILE_PATH_1.tiff"` if we are commenting tiff?

Reply:
The image can be any Image format not only tiff.

Reply:
`// Creates an AnyBitmap with multiple image pages from an array` ?